Vector/Model/Room/RoomPreviewData.m

```diff
@@ -43,22 +43,18 @@ - (instancetype)initWithRoomId:(NSString *)roomId emailInvitationParams:(NSDicti
     return self;
 }
 
-- (void)fetchPreviewData:(void (^)(BOOL))completion
+- (void)peekInRoom:(void (^)(BOOL successed))completion
 {
-    // Make an /initialSync request to get preview data
-    [_mxSession.matrixRestClient initialSyncOfRoom:_roomId withLimit:0 success:^(MXRoomInitialSync *roomInitialSync) {
+    [_mxSession peekInRoomWithRoomId:_roomId success:^(MXPeekingRoom *peekingRoom) {
 
-        _roomState = [[MXRoomState alloc] initWithRoomId:_roomId andMatrixSession:_mxSession andDirection:YES];
+        // Create the room data source
+        // It will be automatically released in the destroy metho of the RoomViewController
+        // that will display the data source
```

Comment:
Replace this comment with "FIXME: release this room data source like it should be"